security/tailscale/src/opnsense/service/templates/OPNsense/Tailscale/rc.conf.d

```diff
@@ -0,0 +1,46 @@
+# DO NOT EDIT
+# THIS FILE IS AUTOMATICALLY GENERATED - ANY CHANGES WILL BE OVERWRITTEN
+#
+{%  if not helpers.empty('OPNsense.tailscale.settings.enabled')  %}
+tailscaled_enable="YES"
```

Comment:
```suggestion
tailscaled_enable="YES"
tailscaled_env="TS_DEBUG_NETSTACK_SUBNETS=0"
```

can you test this one?

Reply:
with this enabled, can no longer access subnets behind opnsense from another tailscale device

Reply:
currently in template but commented out as it breaks, with a link to github thread